tests/inspector_test.py

```diff
@@ -0,0 +1,12 @@
+from pyxnat import Interface
+import os.path as op
+from . import skip_if_no_network
+
+central = Interface(config=op.join(op.dirname(op.abspath(__file__)), 'central.cfg'))
+
+def test_inspector_structure():
+    from pyxnat.core import Inspector
+    i = Inspector(central)
+    i.set_autolearn()
+    print(i.datatypes())
+    s = i.structure()
```

Comment:
Function `structure()` does not return any value just prompts hierarchy of XNAT entities

Reply:
Yes. Mostly because what that function `structure()` is printing out something. Does not return anything, therefore that is definitely not the best possible test in the world... Ideally it would have to check the printed output, or maybe by changing the initial function, making it return a string (instead of printing it) and then comparing the string to a given value. Or leave it like this at the moment...

Reply:
This `structure()` functionality is not of great interest, however, from a `best practices` point of view it would be better to modify its guts as stated. We can perhaps add it as an issue and deal with it later on?

Reply:
see https://github.com/pyxnat/pyxnat/issues/122